src/python/pants/notes/1.26.x.rst

```diff
@@ -3,7 +3,16 @@
 
 This document describes releases leading up to the ``1.26.x`` ``stable`` series.
 
-The ``1.26.x`` series adds support for TOML config files, e.g. ``pants.toml``. INI config files will still be supported for some time, but INI is now legacy. Please see https://groups.google.com/forum/#!topic/pants-devel/N1H03oJONco for more information.
+Some significant changes in this series:
+
+* Pants now uses PEX ``2.1.x``, which uses PIP under the hood, and consequently significantly changes resolves (in order to better align with the wider Python ecosystem). Please see https://github.com/pantsbuild/pex/pull/788 for details.
```

Comment:
Awesome. Thanks for adding this. I'd love for us to start putting more executive summaries like this at the top of release notes.

I was talking to a user this morning that it's hard for them to figure out when upgrading from, say, 1.24.x to 1.27.x, what the main changes are and why they should upgrade.